frontend/app/styles/secret-input.scss

```diff
@@ -2,10 +2,10 @@
 
 @font-face {
   font-family: "text-security-disc";
-  src: url("/text-security-disc-compat.eot");
-  src: url("/text-security-disc-compat.eot?#iefix") format("embedded-opentype"),
-    url("/text-security-disc.woff2") format("woff2"),
-    url("/text-security-disc-compat.ttf") format("truetype");
+  src: url("http://localhost:3000/text-security-disc-compat.eot");
```

Comment:
I don't know much about CORS but I can't imagine this working on anything but a local development setup.

It works locally, but I hardly doubt that it will once it is deployed since we're just hardcoding the local url into this... Maybe someone else has to look into this :sweat_smile: